db/migrate/20210513222021_create_pets.rb

```diff
@@ -0,0 +1,22 @@
+class CreatePets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :pets do |t|
+      t.references :user, foreign_key: true
+      t.string :name
+      t.integer :age
+      t.integer :type
```

Comment:
Per our conversation, and just so its documented.
we should make 1 (other) the default value for the type